incubator/hnc/pkg/controllers/object_controller.go

```diff
@@ -325,6 +337,23 @@ func (r *ObjectReconciler) isExcluded(log logr.Logger, inst *unstructured.Unstru
 	}
 }
 
+// updateReconcileCounter increments both total and current number of reconciles before
+// Reconcile and decrements the current reconciles after Reconcile.
+func (r *ObjectReconciler) updateReconcileCounter(log logr.Logger, ns string, before bool) {
```

Comment:
Does this need to be part of `ObjectReconciler`? It doesn't appear to access any of its members.